runtime/onert/backend/cpu/ops/DepthwiseConvolutionLayer.cc

```diff
@@ -169,11 +169,11 @@ void DepthwiseConvolutionLayer::run()
   }
   else if (_input->data_type() == OperandType::QUANT_UINT8_ASYMM)
   {
-    convQuant8();
+    convQ8uPerTensor();
```

Comment:
In next PR, there will be `if` statement to check per-channel or per-tensor as I did for `conv2d`.